doc/tutorials/app/orbbec_astra_openni.markdown

```diff
@@ -69,6 +69,7 @@ In order to use the Astra camera's depth sensor with OpenCV you should do the fo
     chmod a+r $OUT_FILE
     echo "exit"
     @endcode
+    If you want to use the older SDK version 2.3.0.63, it can still be accessed from here <https://3dclub.orbbec3d.com/t/universal-download-thread-for-astra-series-cameras/622>.
```

Comment:
I propose to rewrite the part and point, that latest version of SDK does not work with modern Linux/ Latest known good version is published on the Orbbec forum ....